client/jetpack-connect/woo-dna-config.js

```diff
@@ -20,9 +20,12 @@ export default ( query ) =>
 		getFlowName: () => 'woodna:' + query.from,
 
 		/**
-		 * @returns {string} Name of the service/plugin the user is signing up for. For example, "WooCommerce Payments".
+		 * @returns {string} Name of the service/plugin the user is signing up for. For example, "WooPayments".
 		 */
-		getServiceName: () => query.woodna_service_name,
+		getServiceName: () =>
```

Comment:
this is shown under the jetpack header and it's retrieved from fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Sjbb%2Qqan.cuc%3Se%3Q85r249rr%2314-og. And called here fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Spynff.wrgcnpx%2Qfreire%2Qirefvba.cuc%3Se%3Qp86809os%231805-og

It's easier to change it here, but probably right way is to change it with a separate diff, but it's a bit risky (it's used in two more places fbhepr%2Sfrnepu%3Sshyy%3QJbb_QAN%253N%253Ntrg_freivpr_anzr%26cebwrpg%3Qjcpbz-og).
What do you think?

Reply:
🤔 Not sure about the best way to handle this either. We could probably do something different on the PR to use the new Jetpack UI from Core.

But we should try to ensure the new name is used everywhere, even on internal code. Although that can be quite hard, knowing that we need to keep it compatible with older versions of the plugin.